index.bs

```diff
@@ -1526,6 +1536,29 @@ When an [=XR input source=] |source| for {{XRSession}} |session| ends its [=prim
 
 Sometimes platform-specific behavior can result in a [=primary action=] being interrupted or cancelled. For example, a [=/XR input source=] may be removed from the [=XRSession/XR device=] after the [=primary action=] is started but before it ends.
 
+Each [=XR input source=] MAY define a <dfn>primary squeeze action</dfn>. The [=primary squeeze action=] is a platform-specific action that, when engaged, produces {{XRSession/squeezestart}}, {{XRSession/squeezeend}}, and {{XRSession/squeeze}} events. The [=primary squeeze action=] should be used for actions rougly mapping to squeezing or grabbing. Examples of possible [=primary squeeze action=]s are pressing a grip trigger or making a grabbing hand gesture. If the platform guidelines define a recommended primary squeeze action then it should be used as the [=primary squeeze action=], otherwise the user agent is free to select one.
```

Comment:
>Each [=XR input source=] MAY define 

Why is this event "MAY" and the other "SHOULD"?

Reply:
We require a primary action, but I don't think we need to require a primary squeeze action.

Reply:
Does that mean the trigger action should be MUST?

Reply:
Hmm. Come to think of it, I thought it already was MUST somewhere in the spec. I don't mind changing it here, given that we're kinda requiring this in the gamepads spec.